auto_submit/lib/service/validation_service.dart

```diff
@@ -146,20 +145,112 @@ class ValidationService {
       }
     }
     // If we got to this point it means we are ready to submit the PR.
-    bool processed = await processMerge(config, result, messagePullRequest);
-    if (processed) await pubsub.acknowledge('auto-submit-queue-sub', ackId);
-    log.info('Ack the processed message : $ackId.');
+    await processMergeSafely(
+        config, gitHubService, messagePullRequest, result, pubsub, ackId, slug, prNumber, Config.kAutosubmitLabel);
+  }
+
+  /// The logic for processing a revert request and opening the follow up
+  /// review issue in github.
+  Future<void> processRevertRequest(
+    Config config,
+    QueryResult result,
+    gh.PullRequest messagePullRequest,
+    String ackId,
+    PubSub pubsub,
+  ) async {
+    ValidationResult revertValidationResult = await revertValidation!.validate(result, messagePullRequest);
+
+    gh.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
+    final int prNumber = messagePullRequest.number!;
+    final GithubService githubService = await config.createGithubService(slug);
+
+    if (revertValidationResult.result) {
+      try {
+        bool processed = await processMerge(config, result, messagePullRequest);
+        if (processed) {
+          gh.Issue issue = await githubService.createIssue(
+            slug,
```

Comment:
Flutter only creates issues on flutter/flutter. This will need to be hard coded instead of dynamic to the repo-under-test

Reply:
OOOh good catch.